source/apiVolontaria/volunteer/admin.py

```diff
@@ -7,9 +7,31 @@ class ParticipationAdmin(admin.ModelAdmin):
     search_fields = ['user__username']
     list_display = ['user', 'standby', 'start_date', 'end_date', 'cell']
 
+    list_filter = [
+        'event__start_date',
+        'event__cycle__name',
+        'event__task_type',
+    ]
+    date_hierarchy = 'event__start_date'
```

Comment:
This part of code have no link with event admin panel. 
Please do not change part of the system without having an issue associated.

Reply:
Do you want me to rephrase my [issue](https://github.com/Volontaria/API-Volontaria/issues/145) instead ?  I thought it was good...

Reply:
Yes, still getting used to git, sorry...